support-frontend/assets/helpers/formValidation.js

```diff
@@ -61,6 +61,16 @@ export const checkStateIfApplicable: ((string | null), CountryGroupId, Contribut
 ) => {
   if (contributionType !== 'ONE_OFF' && (countryGroupId === UnitedStates || countryGroupId === Canada)) {
     return checkBillingState(billingState);
+  } else if (countryGroupId === AUDCountries) {
```

Comment:
contributionType !== 'ONE_OFF' should have also applied here. This caused Single AU Contributions to break. cc @JoemG @rupertbates

Reply:
Fix here: https://github.com/guardian/support-frontend/pull/2400